src/test/java/org/kairosdb/datastore/DatastoreTestHelper.java

```diff
@@ -681,6 +683,7 @@ public void test_queryWithUnicode() throws DatastoreException
 		}
 	}
 
+    @Ignore
```

Comment:
formatting looks broken

Reply:
Possibly - it's inconsistent all over the codebase - I'll take a look.